ariksan.xml

```diff
@@ -679,17 +667,13 @@
 							</treasure>
-							<junk>
-								<de>eine alte Zeitung|ein kaputtes Möbelstück|nicht übereinstimmende Socken|abgelaufene Konserven|einen zerbrochenen Spiegel|ein rostiges Fahrrad|abgetragene Schuhe|eine verstaubte Schallplatte|ein veraltetes VHS-Band|eine abgesplitterte Schale|ein verblichenes Gemälde|ein veralteter Elektroartikel|verknotete Kabel|eine leere Plastikflasche|ein stumpfes Küchenutensil|ein veraltetes Kassettenband|beschädigtes Gepäck|eine zerkratzte CD|eine veraltete Computer-Tastatur|zerrissene Kleidung|kaputtes Spielzeug|ein ausgefranstes Stromkabel|eine durchgelegene Matratze|ein leeres Glasgefäß|eine gesprungene Keramikfigur|abgelaufene Medizin|ein veraltetes Magazin|ein ungenutztes Trainingsgerät|einen zerbrochenen Bilderrahmen|ein ausgetrockneter Stift|verblichene Poster|ein defektes Küchengerät|ein beschädigtes Brettspiel|ein abgenutzter Teppich|Kisten voller abgelaufener Beauty-Produkte|ein veraltetes Mobiltelefon|etwas zersplittertes Glasgeschirr|eine kaputte Uhr|verblichene Weihnachtsdekorationen|ein abgenutztes Kissen|eine gesprungene Porzellanpuppe</de>
-								<en>an old newspaper|a broken furniture|mismatched socks|expired canned food|a cracked mirror|a rusty bicycle|worn-out shoes|a dusty vinyl record|an obsolete VHS tape|a chipped dish|a faded painting|an outdated electronic|tangled cables|an empty plastic bottle|a dull kitchen utensil|an obsolete cassette tape|damaged luggage|a scratched CD|an obsolete computer keyboard|torn clothing|a broken toy|frayed electrical cord|a worn-out mattress|an empty glass jar|a cracked ceramic figurine|expired medicine|an outdated magazine|unused exercise equipment|a broken picture frame|a dried-out pen|faded posters|a defunct kitchen appliance|a damaged board game|a worn-out carpet|boxes full of expired beauty products|an obsolete cellphone|some shattered glassware|a broken clock|faded holiday decorations|a worn-out pillow|a cracked porcelain doll</en>
-							</junk>
 						</variables>
 				</scripttemplate>
 				<scripttemplate index="3" product="2" licence_type="2" guid="90d07145-3341-4917-b20e-4b2d12d37e89">
 					<title>
 						<de>%STATIONMAP:RANDOMCITY%</de>
-						<en>%STATIONMAP:RANDOMCITY%</en>
 					</title>
+					<!--TODO soll treasure hier der gleiche Gegenstand wie in der vorigen Folge sein?-->

```

Comment:
Wenn gleiche Elemente in mehreren Folgen referenziert werden sollen, muss die Variable im Parent definiert werden. Im schlimmsten Fall müsste man ggf. treasure1 und treasure2 einführen, um weiterhin unterschiedliche Dinge im Text haben zu können

Reply:
Hier ist keine Continuity angedacht. Gäbe es eine Option die Episoden aus einem einzigen Child-Template zu generieren (->  script-ariksan-mkw), würde ich es mit diesem Script wahrscheinlich auch tun. In diesem Script ist bloss eine "overarching storyline" drin, die sich aber relativ einfach umschreiben lässt, so dass sie in einer Folge abgehandelt wird.